### PR TITLE
Translate /docs/reference/config-api/kubelet-config.v1alpha1/ page into Chinese

### LINKED ISSUE #32317: [zh]Translate /docs/reference/config-api/kubelet-config.v1alpha1/ page into Chinese
**This is a Feature Request**

[zh]Translate /docs/reference/config-api/kubelet-config.v1alpha1/ page into Chinese

**What would you like to be added**
Add the page /docs/reference/config-api/kubelet-config.v1alpha1 zh localization

**Why is this needed**
Add the page /docs/reference/config-api/kubelet-config.v1alpha1 zh localization
**Comments**
NA

/language zh
/assign

### PR DESCRIPTION
**Pull request by javadoors**
Translate /docs/reference/config-api/kubelet-config.v1alpha1/ page into Chinese #32317

### CODE REVIEW

**Comment by RA489** [comment]
/retitle Translate /docs/reference/config-api/kubelet-config.v1alpha1/ page into Chinese

**Comment by tengqm** [comment]
8 commits for this PR?

/label tide/merge-method-squash

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+auto_generated: true
+-->
+
+<!--
+## Resource Types 
+
+
+- [CredentialProviderConfig](#kubelet-config-k8s-io-v1alpha1-CredentialProviderConfig)
+-->
+## 资源类型 {#kubelet-config-k8s-io-v1alpha1-ResourceTypes}
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:21]:
```suggestion
## 资源类型 {#resource-types}
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+and URL path.</p>
+-->
+</tr>
+<tr><td><code>matchImages</code> <B>[必需]</B><br/>
+<code>[]string</code>
+</td>
+<td>
+   <p>matchImages 是一个必填的字符串列表，用于匹配镜像以确定是否应调用此提供程序。
+如果其中一个字符串与来自 Kubelet 的请求镜像匹配，则将调用插件并有机会提供凭据。
+镜像应包含 Registry 域和 URL 路径。</p>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:146]:
```suggestion
镜像应包含仓库的域名和 URL 路径。</p>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+<!--
+<p>Each entry in matchImages is a pattern which can optionally contain a port and a path.
+Globs can be used in the domain, but not in the port or the path. Globs are supported
+as subdomains like '<em>.k8s.io' or 'k8s.</em>.io', and top-level-domains such as 'k8s.<em>'.
+Matching partial subdomains like 'app</em>.k8s.io' is also supported. Each glob can only match
+a single subdomain segment, so *.io does not match *.k8s.io.</p>
+<p>A match exists between an image and a matchImage when all of the below are true:</p>
+-->
+<p>matchImages 中的每个条目都是一个模式，可以选择包含一个端口和一个路径。
+Glob 可以在域中使用，但不能在端口或路径中使用。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:157]:
```suggestion
域名中可以使用 Glob 模式字符串，但不能在端口或路径中使用。
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+支持 Glob 作为子域，如 “<em>.k8s.io” 或 “k8s.</em>.io”，以及顶级域，如 “k8s.<em>”。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:158]:
```suggestion
使用 Glob 作为子域名也是支持的，如 “*.k8s.io” 或 “k8s.*.io”，以及顶级域，如 “k8s.*”。
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 5]
```diff
@@ -0,0 +1,473 @@
...
+支持匹配部分子域，如 “app</em>.k8s.io”。每个 Glob 只能匹配单个子域段，因此 *.io 不匹配 *.k8s.io。</p>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:159]:
```suggestion
支持匹配部分子域，如 “app*.k8s.io”。每个 Glob 只能匹配单个子域段，因此 '*.io' 不匹配 '*.k8s.io'。</p>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 6]
```diff
@@ -0,0 +1,473 @@
...
+<li>*.azurecr.io</li>
+<li>gcr.io</li>
+<li><em>.</em>.registry.io</li>
+<li>registry.io:8080/path</li>
+</ul>
+</td>
+</tr>
+-->
+<ul>
+<li>两者都包含相同数量的域部分，并且每个部分都匹配。</li>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:180]:
```suggestion
<li>两者都包含相同数量的域名组成部分，并且每个部分都匹配。</li>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 7]
```diff
@@ -0,0 +1,473 @@
...
+<li>imageMatch 的 URL 路径必须是目标镜像 URL 路径的前缀。</li>
+<li>如果 imageMatch 包含一个端口，那么该端口也必须与镜像匹配。</li>
+</ul>
+<p>matchImages 的示例值：</p>
+<ul>
+<li>123456789.dkr.ecr.us-east-1.amazonaws.com</li>
+<li>*.azurecr.io</li>
+<li>gcr.io</li>
+<li><em>.</em>.registry.io</li>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:189]:
```suggestion
<li>*.registry.io</li>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 8]
```diff
@@ -0,0 +1,473 @@
...
+</td>
+</tr>
+</tbody>
+</table>
+-->
+<tr><td><code>json</code> <B>[必需]</B><br/>
+<a href="#JSONOptions"><code>JSONOptions</code></a>
+</td>
+<td>
+   <p>[实验特性] JSON 包含日志格式选项 &quot;json&quot;。</p>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:376]:
```suggestion
   <p>[实验特性] JSON 包含日志格式 &quot;json&quot; 对应的选项。</p>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 9]
```diff
@@ -0,0 +1,473 @@
...
+
+<!--
+<p>JSONOptions contains options for logging format &quot;json&quot;.</p>
+
+
+<table class="table">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
+<tbody>
+-->
+<p>JSONOptions 包含日志格式选项 &quot;json&quot;。</p>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:405]:
```suggestion
<p>JSONOptions 包含日志格式 &quot;json&quot; 对应的选项。</p>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 10]
```diff
@@ -0,0 +1,473 @@
...
+   <p>[Experimental] SplitStream redirects error messages to stderr while
+info messages go to stdout, with buffering. The default is to write
+both to stdout, without buffering.</p>
+</td>
+-->
+<tr><td><code>splitStream</code> <B>[必需]</B><br/>
+<code>bool</code>
+</td>
+<td>
+   <p>[实验特性] SplitStream 将 Error 消息重定向到 stderr，而 Info 消息通过缓冲发送到 stdout。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:426]:
```suggestion
   <p>[实验特性] SplitStream 将错误消息重定向到 stderr，而提示消息通过缓冲发送到 stdout。
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 11]
```diff
@@ -0,0 +1,473 @@
...
+   默认是把 Error 与 Info 消息都没有缓冲的输出到 stdout。</p>
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:427]:
```suggestion
   默认是将错误与提示消息都输出到 stdout，且不提供缓存。</p>
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md (hunk 4 above)
Comment (javadoors) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:158]: 这里是嵌套在html标签里的<em>替换成*感觉是不是有点别扭？

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md (hunk 7 above)
Comment (javadoors) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:189]: What is the reason for the modification here?

**Comment by javadoors** [comment]
I fixed everything, please review,thanks.

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md (hunk 7 above)
Comment (tengqm) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md]: This looks like a generator bug.

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md (hunk 4 above)
Comment (tengqm) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md]: I believed I fixed that bug in the generator, but not sure why there are still issues in parsing the '*' characters.

https://github.com/kubernetes-sigs/reference-docs/pull/270

Comment (tengqm) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md]: Proposed a PR to the upstream repo: https://github.com/kubernetes/kubernetes/pull/108841

**Comment by tengqm** [comment]
Looks like this PR is not needed. Feel free to reopen if there is a different opinion.
/close

**Pull request closed without merging**